css/style.css

```diff
@@ -1,18 +1,334 @@
-.main {
-    display: flex;
+body {
+	box-sizing: content-box;
+	margin: 0;
+	background-color: #f9f9f9;
+	font-family: 'Roboto', sans-serif;
 }
-.navbar{
-    height: 60px;
+
+div, h4, h3, h1{
+	display: block;
+	margin: 0;
+	padding: 0;
+	background: transparent;
 }
-li {
-    list-style-type: none;
-    padding: 10px;
 
+#menu-icon {
+	display: flex;
+	align-items: center;
+	font-size: 1.5rem;
+	padding-left: 1rem;
+}
+
+
+i {
+	color: #606060;
+}
+
+
+
+.navbar {
+	background-color: white;
+	display: flex;
+	font-size: 30px;
+	justify-content: space-between;
+	position: fixed;
+	width: 100%;
+	z-index: 1;
+}
+
+.navbar-items {
+	display: flex;
+	padding: 5px;
+}
+
+.logo {
+	height: 3rem;
+	padding-left: 1rem;
+}
+.notification {
+	/* padding-top: 0.25rem; */
```

Comment:
- [ ] Please, for good practices, do not leave commented pieces of code
![image](https://user-images.githubusercontent.com/53256139/90434121-833ecf00-e092-11ea-89bc-e26601803eee.png)